app/src/pages/project/SpansTable.tsx

```diff
@@ -72,10 +73,66 @@ const PAGE_SIZE = 50;
 
 type RootSpanFilterValue = "root" | "all";
 
+const defaultColumnSettings = {
+  minSize: 100,
+} satisfies Partial<ColumnDef<unknown>>;
+
 function isRootSpanFilterValue(val: unknown): val is RootSpanFilterValue {
   return val === "root" || val === "all";
 }
 
+const TableBody = <T extends { trace: { traceId: string }; id: string }>({
```

Comment:
Direct copy of the previous tbody content, minus new styles in the `td` element to prevent wrapping.